app/pages/Auth/AuthPage.presenter/AuthPage.presenter.tsx

```diff
@@ -84,13 +85,16 @@ export const AuthPage: FC = (): JSX.Element => {
   const [accountIds, setAccountIds] = useState([]);
 
   const offlineStart = localStore.getOfflineStart();
-
   useEffect(() => {
     const controller = new AbortController();
     (async () => {
       try {
         const status = await getWalletStatus();
         setAccountIds(status.accountIds);
+        if (status.accountIds?.length) {
```

Comment:
before we were flashing the "save your entropy" modal for a second or two on wallet unlock. this fixes that and only shows the modal on new account creation